bridge/client/app/_components/ktb-webhook-settings/ktb-webhook-settings.component.ts

```diff
@@ -30,6 +31,8 @@ export class KtbWebhookSettingsComponent implements OnInit {
     pinnable: true,
     originY: 'center',
   };
+  public projectName = this.route.snapshot.paramMap.get('projectName');
```

Comment:
Could you change it to an observable and not use the snapshot here?
```suggestion
  public projectName$ = this.route.paramMap.pipe(map(params => params.get('projectName')));
```